src/lib/core/partialcubeop.test.ts

```diff
@@ -49,241 +74,313 @@ const CUBE_FACE_3X3 = {
   },
 };
 
-describe("3x3 clockwise rotation", () => {
+function expectAllOtherFacesUnaffected(
+  cubeFace: typeof CUBE_FACE_3X3 | typeof CUBE_FACE_2X2,
+  operation: (cubeIndex: CubeIndex, face: Face, dimension: number) => CubeIndex,
+): void {
+  Object.entries(cubeFace)
+    .filter(([key, _]) => key !== "dimension")
```

Comment:
would it make sense to make the test object something like:

```js
{
  dimension: 3,
  facelets: {
    topLeft: {...}
    ...
  }
}
```

so you could just skip this filtering step?

e.g.
```js
Object.entries(cubeFace.facelets).forEach(([key, value]) => {...})
```

(I don't know if facelet is the proper nomenclature here)

also side note, technically in JS, `_` is still a real variable, you should just be able to omit it entirely if you're not using it

Reply:
It would make a lot of sense. Done.